external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DB2IntegrationSuite.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.spark.tags.DockerTest
 @Ignore // AMPLab Jenkins needs to be updated before shared memory works on docker
```

Comment:
Could you update the PR description since this test suite is ignored by the code?

Reply:
Sorry @dongjoon-hyun I didn't realize that `DB2IntegrationSuite` was ignored before.
This seems to be an image issue. It can not start DB2: `/var/db2_setup/lib/setup_db2_instance.sh: line 191: instance_name: parameter null or not set`.

I reverted the upgrade DB2.

Full logs:
```
LM-SHC-16502798:SPARK-28248 yumwang$ docker logs -f d7e35b8964b1
(*) Previous setup has not been detected. Creating the users...
(*) Creating users ...
(*) Creating instance ...
DB2 installation is being initialized.

 Total number of tasks to be performed: 4
Total estimated time for all tasks to be performed: 309 second(s)

Task #1 start
Description: Setting default global profile registry variables
Estimated time 1 second(s)
Task #1 end

Task #2 start
Description: Initializing instance list
Estimated time 5 second(s)
Task #2 end

Task #3 start
Description: Configuring DB2 instances
Estimated time 300 second(s)

Task #3 end

Task #4 start
Description: Updating global profile registry
Estimated time 3 second(s)
Task #4 end

The execution completed successfully.

For more information see the DB2 installation log at "/tmp/db2icrt.log.67".
DBI1446I  The db2icrt command is running.


DBI1070I  Program db2icrt completed successfully.


(*) Enabling TEXT_SEARCH for instance ...
DB2 installation is being initialized.

 Total number of tasks to be performed: 4
Total estimated time for all tasks to be performed: 309 second(s)

Task #1 start
Description: Setting default global profile registry variables
Estimated time 1 second(s)
Task #1 end

Task #2 start
Description: Initializing instance list
Estimated time 5 second(s)
Task #2 end

Task #3 start
Description: Configuring DB2 instances
Estimated time 300 second(s)
Task #3 end

Task #4 start
Description: Updating global profile registry
Estimated time 3 second(s)
Task #4 end

The execution completed successfully.

For more information see the DB2 installation log at
"/tmp/db2iupdt.log.12007".
DBI1446I  The db2iupdt command is running.


DBI1070I  Program db2iupdt completed successfully.


07/05/2019 05:25:35     0   0   SQL1032N  No start database manager command was issued.
SQL1032N  No start database manager command was issued.  SQLSTATE=57019
(*) Cataloging existing databases
ls: cannot access /database/data/db2inst1/NODE0000: No such file or directory
(*) Applying Db2 license ...

LIC1402I  License added successfully.


LIC1426I  This product is now licensed for use as outlined in your License Agreement.  USE OF THE PRODUCT CONSTITUTES ACCEPTANCE OF THE TERMS OF THE IBM LICENSE AGREEMENT, LOCATED IN THE FOLLOWING DIRECTORY: "/opt/ibm/db2/V11.5/license/en_US.iso88591"
(*) Saving the checksum of the current nodelock file ...
(*) Updating DBM CFG parameters ...
DB20000I  The UPDATE DATABASE MANAGER CONFIGURATION command completed
successfully.
DB20000I  The UPDATE DATABASE MANAGER CONFIGURATION command completed
successfully.
DB20000I  The UPDATE DATABASE MANAGER CONFIGURATION command completed
successfully.
No Cgroup memory limit detected, instance memory will follow automatic tuning
(*) Remounting /database with suid...
mount: permission denied

DB2 State : Operable
DB2 has not been started
Starting DB2...

07/05/2019 05:27:01     0   0   SQL1042C  An unexpected system error occurred.
SQL1032N  No start database manager command was issued.  SQLSTATE=57019
(*) Starting TEXT SEARCH service ...
CIE00001 Operation completed successfully.
ssh-keygen: generating new host keys: RSA1 RSA DSA ECDSA ED25519
(*) All databases are now active.
(*) Setup has completed.
/var/db2_setup/lib/setup_db2_instance.sh: line 191: instance_name: parameter null or not set
```